frontends/election-manager/src/screens/smartcards_screen.tsx

```diff
@@ -297,3 +304,28 @@ export function SmartcardsScreen(): JSX.Element {
     </React.Fragment>
   );
 }
+
+export function SmartcardsScreen(): JSX.Element {
+  if (!areVvsg2AuthFlowsEnabled()) {
+    return <LegacySmartcardsScreen />;
+  }
+  return (
+    <NavigationScreen>
+      <Prose maxWidth={false}>
+        <h1>Smartcards</h1>
+        <h2>Election Cards</h2>
+        <p>
+          Insert a card to view card details and/or to create an{' '}
+          <strong>Admin or Poll Worker</strong> card for this election. The
+          election definition must be locked before Admin and Poll Worker cards
```

Comment:
I think the part about locking the election definition would be more useful as a warning. E.g. if the election definition is not locked, this screen could have a warning icon and this message, and also have the buttons/actions disabled.

If the election definition is locked, it doesn't need to say anything - the screen can just work normally.

Reply:
Agreed - I'm all for keeping copy as minimal as necessary and only surfacing extra copy when truly necessary

Reply:
Removed the copy for now and took left a note on https://github.com/votingworks/vxsuite/issues/1917 to add the copy conditionally once locking has been implemented!